### PR TITLE
Add JSON Preview

### PR DESCRIPTION
**Pull request by archturtle**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +32,7 @@ <h2>A small tool for generating and downloading Powercord Manifest Files</h2>
                 <input type="button" value="Plugins" class="selectItem selected" data-type="plugins" />
                 <input type="button" value="Themes" class="selectItem" data-type="themes" />
             </div>
-            <div class="generator selected" data-type="plugins" data-fileName="manifest.json">
```
Comment (ghostlydilemma) [inline index.html]: this is for the download part and should not be changed as manifest files are called like that

File: index.html [hunk 2]
```diff
@@ -105,7 +107,7 @@ <h2>Optional</h2>
                     </div>
                 </form>
             </div>
-            <div class="generator" data-type="themes" data-fileName="powercord_manifest.json">
```
Comment (ghostlydilemma) [inline index.html]: this is for the download part and should not be changed as manifest files are called like that

File: index.html [hunk 3]
```diff
@@ -140,7 +142,7 @@ <h2>Required</h2>
                                     under.</span><br></label><br>
                             <input type="text" name="license" id="pc-themes-license" placeholder=" ">
                         </div>
-                        <div class="entry" data-required="true" data-filled="false">
```
Comment (ghostlydilemma) [inline index.html:143]: Doesn't really need to be removed

**Review by ghostlydilemma — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (archturtle) [reply index.html]: Changed back to original.

File: index.html (hunk 1 above)
Comment (archturtle) [reply index.html]: Changed back to original.

**Review by ghostlydilemma — APPROVED** [review]

**Pull request merged**